snowpack/src/commands/dev.ts

```diff
@@ -387,6 +387,31 @@ export async function startServer(
     }
   }
 
+  function getOutputExtensionMatch() {
+    let outputExts: string[] = [];
+    for (const plugin of config.plugins) {
+      if (plugin.resolve) {
+        for (const outputExt of plugin.resolve.output) {
+          const ext = outputExt.toLowerCase();
+          if (!outputExts.includes(ext)) {
+            outputExts.push(ext);
+          }
+        }
+      }
+    }
+    outputExts = outputExts.sort((a,b) => b.split('.').length - a.split('.').length)
+  
+    return (base: string): string => {
+      const basename = base.toLowerCase();
+      for (const ext of outputExts) {
+        if (basename.endsWith(ext)) return ext;
+      }
+      return path.extname(basename);
+    }
+  }
+  
+  const matchOutputExt = getOutputExtensionMatch();
```

Comment:
nit: this is more of a project style thing than a code thing, but try to avoid abbreviations unless absolutely necessary. Especially in open source, abbreviations may not be obvious to a new reader and can take a while to drop the cognitive overhead.

`Ext` is one of the more harmless of the bunch, but this reminds me of the `BrOrWs...` fn that we were looking at a while ago, and how hard that was to parse as a first time reader :) So its a good habit to get into.